source/common/network/listen_socket_impl.cc

```diff
@@ -46,7 +46,7 @@ void ListenSocketImpl::setupSocket(const Network::Socket::OptionsSharedPtr& opti
 
 UdsListenSocket::UdsListenSocket(const Address::InstanceConstSharedPtr& address)
     : ListenSocketImpl(ioHandleForAddr(Socket::Type::Stream, address, {}), address) {
-  RELEASE_ASSERT(io_handle_->isOpen(), "");
+  RELEASE_ASSERT(io_handle_ && io_handle_->isOpen(), "");
```

Comment:
From the release notes, the impression I have is that this release assert change would be guarded y the runtime feature. But that's not the case here. Presumably that's ok, because this is just downstream of the actual important code which guarded by the feature. is that right?